bip-update-process.md

```diff
@@ -0,0 +1,646 @@
+original author and the BIP editors. If the author does not respond to email in a timely manner, the BIP editors will
+make a unilateral decision (which may be amended should the original author make a delayed reappearance).
+
+## BIP Licensing
+
+### Specification
+
+Each new BIP must identify at least one acceptable license in its preamble. Licenses must be referenced per their
+respective [SPDX License identifier](https://spdx.org/licenses). New BIPs may be accepted with the licenses described
+below.
+
+For example, a preamble might include the following License header:
+
+    License: CC0-1.0
+             GNU-All-Permissive
+
+In this case, the BIP text is fully licensed under both the Creative Commons CC0 1.0 Universal license as well as the
+GNU All-Permissive License, and anyone may modify and redistribute the text provided they comply with the terms of
+*either* license. In other words, the license list is an "OR choice", not an "AND also" requirement.
+
+It is also possible to license source code differently from the BIP text. An optional License-Code header is placed
+after the License header. Again, each license must be referenced by their respective abbreviation given below.
+
+For example, a preamble specifying the optional License-Code header might look like:
+
+    License:         CC0-1.0
+    License-Code:    MIT
+
+In this case, the code in the BIP is not available under CC0-1.0, but is only available under the terms of the MIT
+License.
+
+BIPs are not required to be *exclusively* licensed under approved terms, and may also be licensed under unacceptable
+licenses *in addition to* at least one acceptable license. In this case, only the acceptable license(s) should be listed
+in the License and License-Code headers.
+
+It is recommended that BIPs that include literal code be dual-licensed under the same license terms as the project it
+modifies. For example, literal code intended for Bitcoin Core would ideally be dual-licensed under the MIT license terms
+as well as one of the above with the rest of the BIP text.
+
+In all cases, details of the licensing terms must be provided in the Copyright section of the BIP.
+
+#### Recommended licenses
+
+* BSD-2-Clause: [OSI-approved BSD 2-clause license](https://opensource.org/licenses/BSD-2-Clause)
+* BSD-3-Clause: [OSI-approved BSD 3-clause license](https://opensource.org/licenses/BSD-3-Clause)
+* CC0-1.0: [Creative Commons CC0 1.0 Universal](https://creativecommons.org/publicdomain/zero/1.0/)
+* GNU-All-Permissive: [GNU All-Permissive License](http://www.gnu.org/prep/maintain/html_node/License-Notices-for-Other-Files.html)
+
+#### Not recommended, but acceptable licenses[^licenses]
+
+* CC-BY-4.0: [Creative Commons Attribution 4.0 International](https://creativecommons.org/licenses/by/4.0/)
+* CC-BY-SA-4.0: [Creative Commons Attribution-ShareAlike 4.0 International](https://creativecommons.org/licenses/by-sa/4.0/)
+* MIT: [Expat/MIT/X11 license](https://opensource.org/licenses/MIT)
```

Comment:
I think there should be either
 - no distinction between recommended and not recommended at all,
 - or a rationale for why some are recommended and some are not.  (A possible rationale could be that even CC-BY(-SA) can be annoying in practice, e.g., if people copy sections/tables from BIPs into source files.) 

In any case, I think that the MIT is, for most purposes, virtually equivalent to BSD, and I don't see a reason to recommend one over the other, except maybe license proliferation. (But then, it's strange anyway that BIPs are often BSD but code in the ecosystem is often MIT, so you can make a point for either side...)